src/Cake.Common/Tools/DNX/Run/DNXRunSettings.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.DNX.Run
+{
+    /// <summary>
+    /// Contains settings used by <see cref="DNXRunner"/>
+    /// </summary>
+    public class DNXRunSettings : ToolSettings
+    {
+        /// <summary>
+        /// Gets or sets the project dir to be used (see dnx --project option)
+        /// </summary>
+        public string Project { get; set; }
+
+        /// <summary>
+        /// Gets or sets the framework to be used when running dnx
+        /// </summary>
+        public string Framework { get; set; }
+
+        /// <summary>
+        /// Gets or sets the configuration to be used when running dnx (see dnx --configuration option)
+        /// </summary>
+        public string Configuration { get; set; }
+
+        /// <summary>
+        /// Gets or sets the app base dir to be used by dnx (see dnx --appbase option)
+        /// </summary>
+        public string AppBase { get; set; }
+
+        /// <summary>
+        /// Gets or sets the lib dir to be used by dnx (see dnx --lib option)
+        /// </summary>
+        public string Lib { get; set; }
+    }

```

Comment:
Packages missing and should be added as an DirectoryPath
```
--packages <PACKAGE_DIR>         Directory containing packages
```
And then to the [runner](https://github.com/cake-build/cake/pull/591/files#diff-e6b73337b5132df2f2fffdd691597c7fR95)